evergreen/mongosqltranslate.yml

```diff
@@ -43,91 +43,91 @@ functions:
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        remote_file: mongosql-rs/artifacts/${version_id}/mongosqltranslate-macos/libmongosqltranslate.dylib
-        local_file: mongosql-rs/release/mongosqltranslate-macos/libmongosqltranslate.dylib
+        remote_file: mongosql-rs/artifacts/${version_id}/mongosqltranslate-macos/libmongosqltranslate-signed.dylib
+        local_file: mongosql-rs/release/mongosqltranslate-macos/libmongosqltranslate-signed.dylib
```

Comment:
If we change all these names, we'll have to make sure the drivers can still access the library. For example, we have this in the odbc driver:
```
  "fetch mongosqltranslate":
    - command: s3.get
      params:
        working_dir: mongosql-odbc-driver
        aws_key: ${release_aws_key}
        aws_secret: ${release_aws_secret}
        local_file: mongosql-odbc-driver/target/release/${lib_prefix}mongosqltranslate.${ext}
        remote_file: mongosqltranslate/${lib_prefix}mongosqltranslate-v${LIBMONGOSQLTRANSLATE_VERSION}-${platform}-${arch}.${ext}
        bucket: translators-connectors-releases
```

Reply:
I seem to remember that we specifically agreed not to change where we copy the file. Let's check again with the team on Tuesday, but I am 99% certain

Reply:
The linux libraries will change no matter what since the file type changes: `libmongosqltranslate.so.sig`.